prettier/entrypoint.sh

```diff
@@ -0,0 +1,16 @@
+#!/bin/bash
+
+set -euo pipefail
+
+# shellcheck disable=SC1091
+source /lib.sh
+
+lint() {
+	git --no-pager diff origin/master --name-only | xargs npx prettier --check
```

Comment:
With `origin/master` hardcoded, we would not be able to use this event for PRs pointing to different branches, or when the default branch is not `master`.

It might be useful to use the `base_ref` from the [payload](https://developer.github.com/v3/activity/events/types/#pushevent) here, with a fallback to `origin/master`

Reply:
Will do. Any objects when i move the code to `lib`? The new method would return the result of the `diff` command.

Also i would like to propose to pass the changed file list to the `lint` and `fix` methods as a parameter. This way other linters could profit to only work on the changed files. What do you think?

Reply:
Scratch the idea of passing the list of changed files to `lint` and `fix` as argumeents. It is  to much hassle with bash. The output of the diff is on stdout requiring to go through various bash hoops to capture that and pass it without any issues. The gain does not justify the work ;)

I'll add this new function, which can be called inside of the `lint`/`fix` method to get the changed files: `_git_changed_files | xargs npx prettier --check`

    _git_changed_files() {
        local -r base_ref="$(jq --raw-output '.base_ref // "origin/master"' "${GITHUB_EVENT_PATH}")"

        git --no-pager diff "${base_ref}" --name-only
    }